plugins/importexport/csv/sample.csv

```diff
@@ -0,0 +1,2 @@
+pressPath,authorString,title,abstract,seriesPath,year,isEditedVolume,locale,filename,doi,keywords,subjects,bookCoverImage,bookCoverImageAltText,categories
+leo,Author1 Surname1<author@pkp.sfu.ca>;Author2 Surname2<author2@sfu.pkp.ca>,Title text,Abstract text,,2024,1,en_US,submission.pdf,https://doi.org/10.1111/hex.12487,keyword1;keyword2;keyword3,subject1;subject2,coverImage.png,"Alt text, with commas",Category 1;Category 2;Category 3
```

Comment:
You can remove the "https://doi.org/" part of the DOI here as it's not part of the identifier itself